pages/api/token/temporary.ts

```diff
@@ -49,18 +50,26 @@ const handler = async (
 ) => {
   try {
     const formID = bearerTokenPayload.formID;
+
     if (formID == undefined) {
       return res.status(403).json({ error: "Invalid form request." });
     }
     if (!process.env.TOKEN_SECRET) {
       return res.status(403).json({ error: "Invalid request." });
     }
+
     const temporaryToken = createTemporaryToken(email, process.env.TOKEN_SECRET);
     await updateTemporaryToken(temporaryToken, email, formID);
+    await sendTemporaryTokenByEmail(email, temporaryToken);
+
     logMessage.info(`Temporary Token Requested: Form ID: ${formID} Email: ${email}`);
     res.status(200).json({ message: "success" });
-  } catch (err) {
-    res.status(500).json({ error: "Malformed API Request" });
+  } catch (error) {
+    if (error.message.includes("Could not send temporary token by email")) {
+      res.status(503).json({ error: "GC Notify service failed to send temporary token" });
```

Comment:
I would expect the status code here to be 500. 503 would indicate that our server is not functioning and not able to handle requests as opposed to something going wrong unexpectedly. In this case if GC Notify doesn't work it would be something went wrong as opposed to our server not being able to handle requests

Reply:
Just followed what was requested in the acceptance criteria list https://github.com/cds-snc/platform-forms-client/issues/558 but you are right 500 is probably more appropriate :)

Reply:
Whoops that was probably a typo then on my part